src/components/Flag/Flag.tsx

```diff
@@ -45,14 +51,14 @@ const Flag = ({ size = 'small', isoCode, countryName = '' }: FlagProps) => {
         <Styled.Picture>
           <source
             type="image/webp"
-            srcSet={`https://flagcdn.com/w${width}/${isoCodeFormatted}.webp`}
+            srcSet={`${cdnLocation}/w${width}/${isoCodeFormatted}.webp`}
           />
           <source
             type="image/png"
-            srcSet={`https://flagcdn.com/w${width}/${isoCodeFormatted}.png`}
+            srcSet={`${cdnLocation}/w${width}/${isoCodeFormatted}.png`}
```

Comment:
I think it would be better to customize the all url. 

Something like : 
```tsx
<MuiTelInput flagUrls={({isoCode, size}) => {
   return {
      png: 'yourUrl' // can be a local file,
     webp: '....',
     jpg: '....'
   }
}} />
```

And you call `flagUrls` function for each Flag.

WDYT ?

Reply:
I like this suggestion! I'm going to implement this :)